stack/dashboard/rpm/docker/x86_64/builder.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
This file should not be here

Reply:
Done in [37aaf0d](https://github.com/wazuh/wazuh-packages/pull/1904/commits/37aaf0d5a78129b37a4aef680fa28d87079887a9)